mongoid_lazy_migration.gemspec

```diff
@@ -13,9 +13,12 @@ Gem::Specification.new do |s|
   s.summary     = "Mongoid lazy migration toolkit"
   s.description = "Migrate your documents lazily in atomic, or locked fashion to avoid downtime"
 
-  s.add_dependency("mongoid", "5.0.1")
+  s.add_dependency("mongoid")
```

Comment:
Well, can I use this gem with Mongoid 4 or Mongoid 6? :)
I would prefer to use something like `s.add_dependency("mongoid",  "~> 5.0")`, it is still up to application to lock down the version, but you make sure nobody uses that with totally incompatible version :)